airbyte-integrations/connectors/source-onesignal/Dockerfile

```diff
@@ -34,5 +34,5 @@ COPY source_onesignal ./source_onesignal
 ENV AIRBYTE_ENTRYPOINT "python /airbyte/integration_code/main.py"
 ENTRYPOINT ["python", "/airbyte/integration_code/main.py"]
 
-LABEL io.airbyte.version=0.1.2
+LABEL io.airbyte.version=0.1.3
```

Comment:
> Due to changes on Onesignal side, we cannot retrieve application keys no more (empty fields in response).
User has to fill in application data to retrieve records from app-related streams. (see screen)

I think this makes this a breaking change?